### PR TITLE
Added writedoc command

### PR DESCRIPTION
**Pull request by Samyocord**
Hey, I've added a command to write a document in full screen. With new lines, etc.

### CODE REVIEW

File: coldconsole.py [hunk 1]
```diff
@@ -1,297 +1,340 @@
...
+        f.close()
+
+# clear screen
+def clear():
+    if ostype != "Windows":
+        os.system("clear")
+    else:
+        os.system("cls")
+
+def writedoc():
```
Comment (edgelord314) [inline coldconsole.py]: There should be an argument that specifies the file name, like -f FILE

File: coldconsole.py [hunk 2]
```diff
@@ -1,297 +1,340 @@
...
+    "sysinfo": "sysinfo()",
+    "CUN": "i = input('Name: ');CUN(i)",
+    "mkfile": "mkfile()",
+    "ls": "ls()",
+    "del": "DEL()",
+    "clear": "clear()",
+    "shutdown": "leave();exit()",
+    "reboot": "leave();Boot()",
+    "man": "man()",
+    "writedoc": "writedoc()"
```
Comment (edgelord314) [inline coldconsole.py]: Don't really like that name, it should be something shorter, like edit, but then it also be able to open a existing file for editing, which would be really nice

**Comment by edgelord314** [comment]
Like the idea and the effort, good job

File: coldconsole.py (hunk 1 above)
Comment (Samyocord) [reply coldconsole.py]: After you wrote *exit, it asks for a file name and for a disk name

Comment (edgelord314) [reply coldconsole.py]: Sure, saw it, but it would be nicer (and more fitting) if it would take that in as arguments

**Pull request merged**